client/core/trade_simnet_test.go

```diff
@@ -383,6 +384,11 @@ func TestMakerGhostingAfterTakerRedeem(t *testing.T) {
 			client.log("tick failure: %v", err)
 		}
 
+		// Propagation to miners can take some time after the send RPC
+		// completes, especially with SPV wallets, so wait a bit before mining
+		// blocks in monitorTrackedTrade.
+		time.Sleep(sleepFactor * time.Second)
```

Comment:
Does this only affect this test? I think I noticed it somewhere else and added this sleep to before the `tmuxRun` in `mineBlocks`.

Reply:
Ah, I added it in `monitorTrackedTrade` just before `mineBlocks` as well.
 https://github.com/decred/dcrdex/pull/1265/files#diff-a759d94cd368816442988c181142442065b32de45d225ac4a4da1ab9fe2ac44dR1102

So redundant.  Can also go in `mineBlocks` too suppose.  I had that at some point in the past too.

Let me know what you want to do with this PR and #788, but TBH I'd really like if 788 had anything useful from this PR.  Seems like you've addressed a number of issues with #788, and even made the maker/taker randomness a moot point with AuditContract not even searching the chain.

Reply:
I can cherry-pick this commit into #788, if that works for you.

Reply:
OK.  I wasn't planning to fix the resend pending requests or order status reconciliation test in this one though, just the trade tests so that we know that btc spv is fine before we merge dcr spv.

So I'll just merge this PR and you can reshape it as you like for 788.